### PR TITLE
Fix remote install issue

### PR DESCRIPTION
**Pull request by pjenys0**
- Support transfer local installer than download from pastel website
- Add setup guideline for steps need to be executing in sudo permission - will improve it later
- Add  log messages of remote command to log

### CODE REVIEW

File: cmd/install.go [hunk 1]
```diff
@@ -205,40 +208,59 @@ func runInstallSuperNodeRemoteSubCommand(ctx context.Context, config *configs.Co
...
+			log.WithContext(ctx).WithError(err).Error("Failed to Transferering local Pastel Utility to remote")
+			return err
+		}
 
-	log.WithContext(ctx).Info("Finished Downloading Pastel-Utility Successfully")
+		log.WithContext(ctx).Info("Finished Transferering local Pastel-Utility Successfully")
+	}
 
-	_, err = client.Cmd(fmt.Sprintf("chmod 777 /%s", pastelUtilityPath)).Output()
+	err = client.ShellCmd(ctx, fmt.Sprintf("chmod 777 /%s", pastelUtilityPath))
```
Comment (georgkcode0) [inline cmd/install.go:242]: seem redundant `/`

File: cmd/install.go [hunk 2]
```diff
@@ -675,6 +712,21 @@ func openPort(ctx context.Context, portList []string) (err error) {
 	return nil
 }
 
+func showOpenPortGuideline(ctx context.Context, portList []string) {
+	log.WithContext(ctx).Warn(" - Open ports:")
+
+	for k := range portList {
+		switch utils.GetOS() {
+		case constants.Linux:
+			log.WithContext(ctx).Warnf("   sudo ufw allow %s", portList[k])
```
Comment (georgkcode0) [inline cmd/install.go:721]: space is redundant, doesn't it ?

File: cmd/install.go [hunk 3]
```diff
@@ -675,6 +712,21 @@ func openPort(ctx context.Context, portList []string) (err error) {
...
+		case constants.Windows:
+			log.WithContext(ctx).Warnf("   netsh advfirewall firewall add rule name=TCP Port %s dir=in action=allow protocol=TCP localport=%s", portList[k], portList[k])
```
Comment (georgkcode0) [inline cmd/install.go:723]: space is redundant, doesn't it ?

File: cmd/install.go [hunk 4]
```diff
@@ -775,3 +830,16 @@ func installChrome(ctx context.Context, config *configs.Config) (err error) {
 	}
 	return nil
 }
+
+func showInstallChromeGuideline(ctx context.Context, _ *configs.Config) {
+	if utils.GetOS() == constants.Linux {
+		log.WithContext(ctx).Warn(" - Install chrome:")
+		log.WithContext(ctx).Warnf("   wget %s", constants.ChromeDownloadURL[utils.GetOS()])
+		log.WithContext(ctx).Warnf("   sudo dpkg -i %s", constants.ChromeExecFileName[utils.GetOS()])
```
Comment (georgkcode0) [inline cmd/install.go:838]: space is redundant, doesn't it ?

File: cmd/install.go (hunk 2 above)
Comment (pjenys0) [reply cmd/install.go:721]: No, need this indentation spaces to show instruction steps like:
```
- Config ports:
       sudo ufw allow 123
       sudo ufw allow 456
```

File: cmd/install.go (hunk 4 above)
Comment (pjenys0) [reply cmd/install.go:838]: no - same above.

File: cmd/install.go (hunk 1 above)
Comment (nvnguyen202) [reply cmd/install.go:242]: but no impact at all. because this pastelUtilityPath is absolute path. will fix it later in other PR

**Pull request closed without merging**